local-tests/setup/session-sigs/get-pkp-session-sigs.ts

```diff
@@ -8,7 +8,8 @@ import { TinnyPerson } from '../tinny-person';
 export const getPkpSessionSigs = async (
   devEnv: TinnyEnvironment,
   alice: TinnyPerson,
-  resourceAbilityRequests?: LitResourceAbilityRequest[]
+  resourceAbilityRequests?: LitResourceAbilityRequest[],
+  expiration?: string
```

Comment:
Adding this since we need PKP sessionSig expiry for Lambdas to be less than 15 mins